rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
```

Comment:
Getters and Setters for `@book` and `@person` instance variables should be present. Kindly add them to the attr_accessor. 🙂